README.md

```diff
@@ -164,4 +191,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+Chomsky Level 2 because the language requires a list of both terminals and non terminals whose meaning is fixed.
```

Comment:
# -3

It's actually type 3, because it can be implemented using an FSM (because you can always work out what do do next based on the current state and the next token)